test/extensions/filters/listener/http_inspector/BUILD

```diff
@@ -15,6 +15,9 @@ envoy_extension_cc_test(
     name = "http_inspector_test",
     srcs = ["http_inspector_test.cc"],
     extension_name = "envoy.filters.listener.http_inspector",
+    #TODO(davinci26): The test passes on Windows *but* http inspector
+    # relies on Event::FileTriggerType::Edge and we get away with it
+    # because we mock the dispather.
```

Comment:
spelling: dispatcher.

Reply:
Thanks good catch. I improved the comment overall in d540ffd472c7a3ceb9097f1b09b1d16acbf8b4bc. Now it takes into consideration that #12584 is merged